tap_klaviyo/streams.py

```diff
@@ -169,9 +169,17 @@ class FlowsStream(KlaviyoStream):
     name = "flows"
     path = "/flows"
     primary_keys = ["id"]
-    replication_key = None
+    replication_key = "updated"
     schema_filepath = SCHEMAS_DIR / "flows.json"
```

Comment:
Can we update `flows.json` to include the top-level `updated` field?

Reply:
yes, sorry I was out for a bit. I'll update and push.